domains/abwaocode.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "abwaocode",
+        "email": "abwaocode.address"
+    },
+    "record": {
+        "CNAME": "abwaocode.github.io"
```

Comment:
I cannot reach your website, 404 and i see no other repo on your profile with github pages enabled

you gave as preview url https://abwaocode.serv00.net/ and then used github pages which is a different host, in no way github pages cname would work on that other host
anyway for https://abwaocode.serv00.net/ I wouldn't consider that complete